pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -147,14 +150,22 @@ func (c *Check) Validate() error {
 	}
 
 	// frequency must be in [1, 120] seconds
-	if c.Frequency < 1*1000 || c.Frequency > 120*1000 {
+	if c.Type() != CheckTypeTraceroute && (c.Frequency < 1*1000 || c.Frequency > 120*1000) {
```

Comment:
So does Traceroute not have a check frequency setting at all? How often will the traceroute check be run?

Reply:
It does, but I sort of grafted on into a different place. The current frequency/timeout settings are the same for all probe types because the frequency/timeout settings make sense for all of them (1-10 second timeout, 10 second to 2 minute frequency). Traceroute, however, usually takes between 5-30 seconds, so we need to have a different range for those settings